testing/prow/benchmarking.sh

```diff
@@ -0,0 +1,78 @@
+#! /usr/bin/env bash
+
+set -x
+set -o pipefail
+set -o errexit
+set -o nounset
+
+EPOCHS=3
+THRESHOLD=0.05
+MINSR=150.0
+MAXDR=30
+
+assess_benchmark_stats () {
+    artifact_dir=${ARTIFACT_DIR:-"/tmp/ci-artifacts_$(date +%Y%m%d)"}
+    sample_rate=$(cat "${artifact_dir}/benchmarking_run_ssd_sample_rate.log")
+    bench_duration=$(cat "${artifact_dir}/benchmarking_run_ssd_bench_duration.log")
+    if [[ $MINSR -gt $sample_rate ]] ; then
+        echo "Sample rate below minimum expected"
```

Comment:
in the `echo` of this function (or maybe before the tests), can you print the content of `$MINSR ` and `$sample_rate` (and same for the duration)

otherwise, when trouble shooting, you look at the logs, see:
```
ERROR: Sample rate below minimum expected
```
(can you add the `ERROR` prefix) and say: great, it was too low, but how low was it? and what was the limit?

Bash's `set -x` would tell, but better make it obvious and state in plain text what went wrong